api-compact.yaml

```diff
@@ -300,83 +292,6 @@ x-amazon-apigateway-integration-with-enterprise-customer-catalog-uuid-and-course
 endpoints:
   v1:
 
-    # /v1/enterprise_catalogs/
```

Comment:
I don't think we should be stripping out the API gateway V1 endpoints -- instead they should be mapped to new backend endpoints and continue to provide responses according to their current formats/schemas.